core/src/main/java/com/novoda/noplayer/drm/DownloadDrmSessionCreator.java

```diff
@@ -0,0 +1,24 @@
+package com.novoda.noplayer.drm;
```

Comment:
Since all this is ExoPlayer (Not AndroidMediaPlayer) specific , can we move it into exoplayer package ?